src/diffusers/schedulers/scheduling_dpmsolver_multistep.py

```diff
@@ -809,6 +810,9 @@ def step(
                 A current instance of a sample created by the diffusion process.
             generator (`torch.Generator`, *optional*):
                 A random number generator.
+            variance_noise (`torch.FloatTensor`):
+                Alternative to generating noise with `generator` by directly providing the noise for the variance
+                itself. Useful for methods such as [`CycleDiffusion`].
```

Comment:
Is this needed for leditspp ? I'm fine with adding such an input if it's used for leditspp, but not if it's for CycleDiffusion

Reply:
can we use `DPMSolverMultistepInverseScheduler` instead?

Reply:
Yes we can probably use DPMSolverMultistepInverseScheduler since it already has the `variance_noise` paramter.

Reply:
No we cannot easily use `DPMSolverMultistepInverseScheduler`, since we are actually iterating through the timesteps from T -> 0, both during inversion and during generation. 
Also yes that change is required for LEdits itself.